philosophy_of_software_design/README.md

```diff
@@ -1,3 +1,13 @@
 # Philosophy of Software Design
 
 This document is a summary of the book Philosophy of Software Design.
+
+## Chapter 1
+### Introduction
+### (It's All About Complexity)
+
+As a program evolves, complexity inevitably increases and the system becomes harder to understand, slowing down development and leading to bugs.
+Good development tools may help, but building more powerful systems more cheaply will come from simpler designs.
+There are two general approaches to fighting complexity:
+- Making code simpler and more obvious. For example, eliminating special cases or using identifiers in a consistent fashion.
+- Encapsulate complexity, so that programmers can work on a system without being exposed to all of its complexity at once. This approach is called modular design, as the software system is divided up into modules designed to be relatively independent of each other.
```

Comment:
```suggestion
- Encapsulate complexity, so that programmers can work on a system without being exposed to all of its complexity at once. This approach is called **modular design**, as the software system is divided up into modules designed to be relatively independent of each other.
```